src/components/navbar/Nav.jsx

```diff
@@ -0,0 +1,118 @@
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+import logo from '../logo.jpg';
+import './Nav.css';
+
+const Nav = () => {
+  const [isMobile, setIsMobile] = useState(false);
+
+
+  return (
+    <div className="navigation-panel">
+      <header>
+        <div
+          className={
+            isMobile
+              ? 'nav-icons-holder '
+              : 'nav-icons-holder nav-mobile-border'
+          }
+        >
+          <img src={logo} alt="Logo" className="logo" />
+          <div className="nav-btn-holder">
+            <Link to="/login">
+              <button className="button-sign-log" type="submit">
+                Log in
+              </button>
+            </Link>
```

Comment:
- [ ] Can we toggle this button to say `Log out` once the user has signed in? It would be great if we could have a Log out button that removes the JWT token from the `sessionStorage`. what do you think?